components/Note/src/note.css

```diff
@@ -0,0 +1,45 @@
+.denhaag-note {
+  --denhaag-note-icon-color: #4b4b4b;
+  --denhaag-note-icon-fill: #f1f1f1;
+  --denhaag-note-icon-stroke: #4b4b4b;
+
+  color: #4b4b4b;
+  display: flex;
+  flex-direction: row;
+  font-family: var(--denhaag-note-font-family, inherit);
+  font-size: var(--denhaag-note-font-size, 1.125em);
+  font-weight: var(--denhaag-note-font-weight, normal);
+  line-height: var(--denhaag-note-line-height, 1.3);
+  gap: var(--denhaag-note-icon-spacing);
+  padding-block-start: var(--denhaag-note-icon-spacing);
+  padding-block-end: var(--denhaag-note-icon-spacing);
+}
+
+.denhaag-note--info {
```

Comment:
These tokens must come from tokens file.

example:
`--denhaag-note-icon-fill: var(--denhaag-note-info-icon-fill)`
`--denhaag-note-icon-fill: var(--denhaag-note-warning-icon-fill)`

Reply:
I did'nt touch the note.css, besides this is not the posttype label component.